ng2-components/ng2-activiti-analytics/src/components/widgets/duration/duration.widget.html

```diff
@@ -5,6 +5,7 @@
                    type="text"
                    pattern="-?[0-9]*(\.[0-9]+)?"
                    [attr.id]="field.id"
+                   [value]="field.value"
```

Comment:
Yes, we need both. Otherwise I can't unit test the component.

let numberElement: any = element.querySelector('#duration');
expect(numberElement.value).toEqual('0');